resources/user-agents/browsers/opera-mobile/opera-mobile-14-on.json

```diff
@@ -549,36 +568,13 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Chrome/*Safari/* OPR/#MAJORVER#.*",
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*)*Chrome/*Safari/* OPR/#MAJORVER#.*",
           "device": "general Tablet",
           "platforms": [
             "Android_3_2", "Android_3_1", "Android_3_0"
           ]
         },
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Chrome/* Mobile Safari/* OPR/#MAJORVER#.*",
```

Comment:
I removed these sets of patterns because they were causing some device patterns above to be detected with a "general Mobile Phone" device.

I was concerned that this might cause a lot of UAs to not be detected as Opera anymore, so I ran these changes against a couple hundred thousand "top" UAs from the past few months and compared to how `master` parsed them.  Only a couple of cases came up, which I added devices for.